package.json

```diff
@@ -121,31 +122,37 @@
         "eslint": "^8.7.0",
         "eslint-config-airbnb-base": "^15.0.0",
         "eslint-config-prettier": "^8.3.0",
+        "eslint-plugin-import": "^2.25.2",
         "eslint-plugin-prettier": "^4.0.0",
         "eslint-plugin-react": "^7.29.4",
         "husky": "^4.3.0",
         "jest": "^26.6.3",
         "jest-environment-jsdom-sixteen": "^1.0.3",
-        "lodash.debounce": "^4.0.6",
+        "lodash.debounce": "4.0.6",
         "np": "^7.2.0",
         "prettier": "^2.6.2",
         "pretty-quick": "^3.1.0",
-        "react": "^17.0.1",
-        "react-dom": "^17.0.1",
-        "ts-jest": "^24.3.0",
+        "react": "17.0.1",
+        "react-dom": "17.0.1",
+        "require-from-string": "^2.0.2",
+        "ts-jest": "^26.5.6",
         "ts-loader": "^8.4.0",
-        "typescript": "^4.8.4",
-        "web-vitals": "^2.1.4"
+        "tslib": "^2.4.0",
+        "typescript": "4.8.4",
+        "web-vitals": "^2.1.4",
+        "webpack": "^4.0.0"
     },
     "peerDependencies": {
         "@emotion/react": "^11.1.5",
-        "@guardian/consent-management-platform": "^10",
-        "@guardian/source-foundations": "^4.0.2",
-        "@guardian/source-react-components": "^4.0.1",
+        "@guardian/commercial-core": "^4.25.0",
+        "@guardian/consent-management-platform": "^10.5.0",
+        "@guardian/libs": "^9.0.1",
+        "@guardian/source-foundations": "^7.0.1",
+        "@guardian/source-react-components": "^9.0.1",
+        "lodash.debounce": "^4.0.6",
```

Comment:
Is there a way we could have a script that prevents these ever getting out of sync?

For example, [`discussion-rendering`’s dependencies.ts script does that](https://github.com/guardian/discussion-rendering/blob/18d8f936c450428d0c89f56b6055d5f77cdd7c1f/scripts/dependencies.ts).

Reply:
That looks fantastic, are you volunteering @mxdvl ?